wcomponents-core/src/main/java/com/github/bordertech/wcomponents/Margin.java

```diff
@@ -17,6 +17,31 @@
  */
 public class Margin implements Serializable {
 
+	/**
+	 * A margin with a size of Size.ZERO on all sides.
+	 */
+	public static final Margin ZERO = new Margin(Size.ZERO);
```

Comment:
I like the idea of providing the "all" margin objects.

Should we name them to indicate they are for all? Like `ZERO_ALL` or something like that....

Also, how about some unit tests for the new all objects.

Reply:
I think the names are Okay as they are.